dashboard/app/helpers/application_helper.rb

```diff
@@ -111,16 +111,15 @@ def external_oauth_sign_out_url(provider)
     end
   end
 
-  def meta_image_url(params)
-    level_source = params[:level_source]
-    if level_source
-      app = level_source.level.game.app
-    else
-      app = params[:app]
+  def meta_image_url
```

Comment:
should I have kept this as passing arguments instead of switching to use the instance variables?